site/docs/v1.2.0/velero-install.md

```diff
@@ -0,0 +1,44 @@
+# Velero Install CLI
+
+This document serves as a guide to using the `velero install` CLI command to install `velero` server components into your kubernetes cluster.
+
+_NOTE_: `velero install` will, by default, use the CLI's version information to determine the version of the server compoents to deploy. This behavior may be overridden by using the `--image` flag. Refer to [Building Server Component Container Images][1].
+
+## Usage
+
+This section explains some of the basic flags supported by the `velero install` CLI command. For a complete explanation of the flags, please run `velero install --help`
+
+```bash
+velero install \
+    --plugins <PLUGIN_CONTAINER_IMAGE [PLUGIN_CONTAINER_IMAGE]>
+    --provider <YOUR_PROVIDER> \
+    --bucket <YOUR_BUCKET> \
+    --secret-file <PATH_TO_FILE> \
+    --velero-pod-cpu-request <CPU_REQUEST> \
+    --velero-pod-mem-request <MEMORY_REQUEST> \
+    --velero-pod-cpu-limit <CPU_LIMIT> \
+    --velero-pod-mem-limit <MEMORY_LIMIT> \
+    [--use-restic] \
+    [--restic-pod-cpu-request <CPU_REQUEST>] \
+    [--restic-pod-mem-request <MEMORY_REQUEST>] \
+    [--restic-pod-cpu-limit <CPU_LIMIT>] \
+    [--restic-pod-mem-limit <MEMORY_LIMIT>]
+```
+
+The values for the resource requests and limits flags follow the same format as [Kubernetes resource requirements][5]
+For plugin container images, please refer to our [supported providers][2] page.
+
+## Examples
+
+This section provides examples that serve as a starting point for more customized installations.
+
+```(bash)
```

Comment:
Parens need to be removed here, but otherwise I think this is ready from my perspective.